Resources/doc/index.markdown

```diff
@@ -4,7 +4,7 @@
 
 ``` yaml
 "require": {
-    "lunetics/locale-bundle": "2.2.*",
+    "lunetics/locale-bundle": "2.4.*",
```

Comment:
yes, `~2.4` is correct right now. but at least when moving to 3.0, it is not anymore. and it will be strange when version 2.5 is stable. so the composer require seems the best way to do this. it also seems to be an emerging best practice to let composer manage the composer.json file rather than editing it by hand.